prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+var number1 = 2;
+var number2 = 4;
+prod = number1 * number2;
```

Comment:
Since `prod` is a new variable that you're creating, it needs a variable initializer like `const`